rmgpy/molecule/adjlist.py

```diff
@@ -787,13 +871,20 @@ def from_adjacency_list(adjlist, group=False, saturate_h=False):
         ConsistencyChecker.check_multiplicity(n_rad, multiplicity)
         for atom in atoms:
             ConsistencyChecker.check_hund_rule(atom, multiplicity)
-        return atoms, multiplicity
+        return atoms, multiplicity, metal, facet
     else:
         # Currently no group consistency check
-        return atoms, multiplicity
+        if not group:
+            n_rad = sum([atom.radical_electrons for atom in atoms])
+            absolute_spin_per_electron = 1 / 2.
+            if multiplicity is None:
+                multiplicity = 2 * (n_rad * absolute_spin_per_electron) + 1
```

Comment:
```suggestion
            if multiplicity is None:
                n_rad = sum([atom.radical_electrons for atom in atoms])
                # absolute_spin_per_electron = 1 / 2.
                # multiplicity = 2 * (n_rad * absolute_spin_per_electron) + 1
                multiplicity = n_rad + 1
```

Reply:
if no need to count the radicals, then don't. And if you do need to, don't multiply and divide by 2 just for fun.

Reply:
Done